packages/ansible-language-server/src/errors.ts

```diff
@@ -246,6 +246,13 @@ ERRORS.addError(
     },
   ),
 );
+ERRORS.addError(
+  404,
+  new Error(
+    "feature_not_available",
+    "The requested action is not yet supported in your environment.",
```

Comment:
`yet` makes it sounds like it's a matter of time. It's not necessarily the case. I would go with a more neutral: `The requested action is not available in your environment.`